### PR TITLE
Allow manual acking

### PR DESCRIPTION
**Pull request by ro0NL**
cc @nekufa 

fixed https://github.com/basis-company/nats.php/issues/27

i think this is the fastest route API-wise :angel:

### CODE REVIEW

**Comment by nekufa** [comment]
hi @ro0NL, thanks for contribution!
Maybe it's better to use reply var name instead of ack on the client side?
For consumer it looks good)

File: src/Client.php [hunk 1]
```diff
@@ -328,8 +328,8 @@ public function process(null|int|float $timeout = 0)
                     }
                     throw new LogicException("No handler for message $message->sid");
                 }
-                $result = $this->handlers[$message->sid]($message->payload);
-                if ($message->replyTo) {
+                $result = $this->handlers[$message->sid]($message->payload, $message->replyTo);
+                if ($reply && $message->replyTo) {
                     $this->send(new Publish([
```
Comment (ro0NL) [inline src/Client.php:333]: should we use `publish()` while at it?

Comment (nekufa) [reply src/Client.php:333]: sounds good! it would make code better

**Comment by ro0NL** [comment]
@nekufa all good

**Comment by nekufa** [comment]
@ro0NL thanks!

**Pull request merged**